2013/02/07 13:13:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "contato".' in /var/www/framework/web/CController.php:483
Stack trace:
#0 /var/www/framework/web/CController.php(270): CController->missingAction('contato')
#1 /var/www/framework/web/CWebApplication.php(282): CController->run('contato')
#2 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/contato')
#3 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/feitobrick/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/contato
HTTP_REFERER=http://localhost:100/menu.html
---
2013/02/07 13:21:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "produtos".' in /var/www/framework/web/CController.php:483
Stack trace:
#0 /var/www/framework/web/CController.php(270): CController->missingAction('produtos')
#1 /var/www/framework/web/CWebApplication.php(282): CController->run('produtos')
#2 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/produtos')
#3 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/feitobrick/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/produtos
HTTP_REFERER=http://localhost:100/index.php?r=site/sobre
---
2013/02/07 13:26:57 [error] [php] htmlspecialchars() expects parameter 1 to be string, array given (/var/www/framework/web/helpers/CHtml.php:98)
Stack trace:
#0 /var/www/framework/web/helpers/CHtml.php(153): renderAttributes()
#1 /var/www/framework/web/helpers/CHtml.php(449): tag()
#2 /var/www/framework/web/helpers/CHtml.php(484): button()
#3 /var/www/feitobrick/protected/views/site/login.php(40): submitButton()
#4 /var/www/framework/web/CBaseController.php(126): require()
#5 /var/www/framework/web/CBaseController.php(95): SiteController->renderInternal()
#6 /var/www/framework/web/CController.php(869): SiteController->renderFile()
#7 /var/www/framework/web/CController.php(782): SiteController->renderPartial()
#8 /var/www/feitobrick/protected/controllers/SiteController.php(100): SiteController->render()
#9 /var/www/framework/web/actions/CInlineAction.php(49): SiteController->actionEntrar()
#10 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams()
#11 /var/www/framework/web/CController.php(286): SiteController->runAction()
#12 /var/www/framework/web/CController.php(265): SiteController->runActionWithFilters()
#13 /var/www/framework/web/CWebApplication.php(282): SiteController->run()
#14 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController()
#15 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#16 /var/www/feitobrick/index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=site/entrar
in /var/www/feitobrick/protected/views/site/login.php (40)
in /var/www/feitobrick/protected/controllers/SiteController.php (100)
in /var/www/feitobrick/index.php (13)
2013/02/07 13:37:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "cadastro".' in /var/www/framework/web/CController.php:483
Stack trace:
#0 /var/www/framework/web/CController.php(270): CController->missingAction('cadastro')
#1 /var/www/framework/web/CWebApplication.php(282): CController->run('cadastro')
#2 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/cadastro')
#3 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/feitobrick/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/cadastro
HTTP_REFERER=http://localhost:100/index.php?r=site/contato
---
2013/02/07 13:40:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "cadastro".' in /var/www/framework/web/CController.php:483
Stack trace:
#0 /var/www/framework/web/CController.php(270): CController->missingAction('cadastro')
#1 /var/www/framework/web/CWebApplication.php(282): CController->run('cadastro')
#2 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/cadastro')
#3 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/feitobrick/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/cadastro
HTTP_REFERER=http://localhost:100/index.php?r=site/contato
---
2013/02/07 13:42:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "cadastro".' in /var/www/framework/web/CController.php:483
Stack trace:
#0 /var/www/framework/web/CController.php(270): CController->missingAction('cadastro')
#1 /var/www/framework/web/CWebApplication.php(282): CController->run('cadastro')
#2 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/cadastro')
#3 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/feitobrick/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/cadastro
HTTP_REFERER=http://localhost:100/index.php?r=site/contato
---
2013/02/07 13:42:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "cadastro".' in /var/www/framework/web/CController.php:483
Stack trace:
#0 /var/www/framework/web/CController.php(270): CController->missingAction('cadastro')
#1 /var/www/framework/web/CWebApplication.php(282): CController->run('cadastro')
#2 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/cadastro')
#3 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/feitobrick/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/cadastro
HTTP_REFERER=http://localhost:100/index.php?r=site/entrar
---
2013/02/07 13:47:45 [error] [exception.CException] exception 'CException' with message 'CWebUser and its behaviors do not have a method or closure named "isGuest".' in /var/www/framework/base/CComponent.php:265
Stack trace:
#0 /var/www/feitobrick/protected/views/layouts/main.php(50): CComponent->__call('isGuest', Array)
#1 /var/www/feitobrick/protected/views/layouts/main.php(50): CWebUser->isGuest()
#2 /var/www/framework/web/CBaseController.php(126): require('/var/www/feitob...')
#3 /var/www/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/feitob...', Array, true)
#4 /var/www/framework/web/widgets/CContentDecorator.php(76): CBaseController->renderFile('/var/www/feitob...', Array, true)
#5 /var/www/framework/web/widgets/CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#6 /var/www/framework/web/widgets/COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#7 /var/www/framework/web/CBaseController.php(206): COutputProcessor->run()
#8 /var/www/framework/web/CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#9 /var/www/feitobrick/protected/views/layouts/column1.php(6): CBaseController->endContent()
#10 /var/www/framework/web/CBaseController.php(126): require('/var/www/feitob...')
#11 /var/www/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/feitob...', Array, true)
#12 /var/www/framework/web/CController.php(784): CBaseController->renderFile('/var/www/feitob...', Array, true)
#13 /var/www/feitobrick/protected/controllers/SiteController.php(30): CController->render('index')
#14 /var/www/framework/web/actions/CInlineAction.php(49): SiteController->actionIndex()
#15 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#16 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#17 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 /var/www/framework/web/CWebApplication.php(282): CController->run('')
#19 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('')
#20 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#21 /var/www/feitobrick/index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/index.html
HTTP_REFERER=http://localhost:100/index.php?r=site/produtos
---
2013/02/07 13:53:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "cadastro".' in /var/www/framework/web/CController.php:483
Stack trace:
#0 /var/www/framework/web/CController.php(270): CController->missingAction('cadastro')
#1 /var/www/framework/web/CWebApplication.php(282): CController->run('cadastro')
#2 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/cadastro')
#3 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/feitobrick/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/cadastro
HTTP_REFERER=http://localhost:100/index.php?r=site/entrar
---
2013/02/07 13:53:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "cadastro".' in /var/www/framework/web/CController.php:483
Stack trace:
#0 /var/www/framework/web/CController.php(270): CController->missingAction('cadastro')
#1 /var/www/framework/web/CWebApplication.php(282): CController->run('cadastro')
#2 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/cadastro')
#3 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/feitobrick/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/cadastro
HTTP_REFERER=http://localhost:100/index.php?r=site/cadastro
---
2013/02/07 13:56:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "cadastro".' in /var/www/framework/web/CController.php:483
Stack trace:
#0 /var/www/framework/web/CController.php(270): CController->missingAction('cadastro')
#1 /var/www/framework/web/CWebApplication.php(282): CController->run('cadastro')
#2 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/cadastro')
#3 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/feitobrick/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/cadastro
HTTP_REFERER=http://localhost:100/index.php?r=site/contato
---
2013/02/07 14:10:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "cadastro".' in /var/www/framework/web/CController.php:483
Stack trace:
#0 /var/www/framework/web/CController.php(270): CController->missingAction('cadastro')
#1 /var/www/framework/web/CWebApplication.php(282): CController->run('cadastro')
#2 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/cadastro')
#3 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/feitobrick/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/cadastro
HTTP_REFERER=http://localhost:100/index.php?r=site/contato
---
2013/02/07 14:10:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "cadastro".' in /var/www/framework/web/CController.php:483
Stack trace:
#0 /var/www/framework/web/CController.php(270): CController->missingAction('cadastro')
#1 /var/www/framework/web/CWebApplication.php(282): CController->run('cadastro')
#2 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/cadastro')
#3 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/feitobrick/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/cadastro
HTTP_REFERER=http://localhost:100/index.php?r=site/contato
---
2013/02/07 14:10:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "cadastro".' in /var/www/framework/web/CController.php:483
Stack trace:
#0 /var/www/framework/web/CController.php(270): CController->missingAction('cadastro')
#1 /var/www/framework/web/CWebApplication.php(282): CController->run('cadastro')
#2 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/cadastro')
#3 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/feitobrick/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/cadastro
HTTP_REFERER=http://localhost:100/index.php?r=site/contato
---
2013/02/07 14:20:49 [error] [exception.CException] exception 'CException' with message 'SiteController and its behaviors do not have a method or closure named "performAjaxValidation".' in /var/www/framework/base/CComponent.php:265
Stack trace:
#0 /var/www/feitobrick/protected/controllers/SiteController.php(47): CComponent->__call('performAjaxVali...', Array)
#1 /var/www/feitobrick/protected/controllers/SiteController.php(47): SiteController->performAjaxValidation(Object(novoUsuarioForm), 'usuario-form')
#2 /var/www/framework/web/actions/CInlineAction.php(49): SiteController->actionNovoUsuario()
#3 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#5 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /var/www/framework/web/CWebApplication.php(282): CController->run('novoUsuario')
#7 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/novoUsuari...')
#8 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#9 /var/www/feitobrick/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=site/novoUsuario
HTTP_REFERER=http://localhost:100/index.php?r=site/novoUsuario
---
2013/02/09 17:01:02 [error] [system.db.CDbCommand] CDbCommand::fetch() failed: SQLSTATE[42501]: Insufficient privilege: 7 ERRO:  permissão negada para relação usuario. The SQL statement executed was: SELECT * FROM "usuario" "t" WHERE email = :id LIMIT 1.
in /var/www/feitobrick/protected/components/UserIdentity.php (8)
in /var/www/feitobrick/protected/models/LoginForm.php (47)
in /var/www/feitobrick/protected/controllers/SiteController.php (121)
2013/02/09 17:01:02 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42501]: Insufficient privilege: 7 ERRO:  permissão negada para relação usuario. The SQL statement executed was: SELECT * FROM "usuario" "t" WHERE email = :id LIMIT 1' in /var/www/framework/db/CDbCommand.php:541
Stack trace:
#0 /var/www/framework/db/CDbCommand.php(414): CDbCommand->queryInternal('fetch', Array, Array)
#1 /var/www/framework/db/ar/CActiveRecord.php(1297): CDbCommand->queryRow()
#2 /var/www/framework/db/ar/CActiveRecord.php(1402): CActiveRecord->query(Object(CDbCriteria))
#3 /var/www/feitobrick/protected/components/UserIdentity.php(8): CActiveRecord->find('email = :id', Array)
#4 /var/www/feitobrick/protected/models/LoginForm.php(47): UserIdentity->authenticate()
#5 /var/www/framework/validators/CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#6 /var/www/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#7 /var/www/framework/base/CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#8 /var/www/feitobrick/protected/controllers/SiteController.php(121): CModel->validate()
#9 /var/www/framework/web/actions/CInlineAction.php(49): SiteController->actionEntrar()
#10 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#11 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#12 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 /var/www/framework/web/CWebApplication.php(282): CController->run('entrar')
#14 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/entrar')
#15 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#16 /var/www/feitobrick/index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/index.php?r=site/entrar
HTTP_REFERER=http://localhost:100/index.php?r=site/entrar
---
2013/02/09 17:04:20 [error] [exception.CException] exception 'CException' with message 'Property "novoUsuarioForm.bairro" is not defined.' in /var/www/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/framework/validators/CStringValidator.php(79): CComponent->__get('bairro')
#1 /var/www/framework/validators/CValidator.php(213): CStringValidator->validateAttribute(Object(novoUsuarioForm), 'bairro')
#2 /var/www/framework/base/CModel.php(159): CValidator->validate(Object(novoUsuarioForm), NULL)
#3 /var/www/feitobrick/protected/controllers/SiteController.php(31): CModel->validate()
#4 /var/www/framework/web/actions/CInlineAction.php(49): SiteController->actionNovoUsuario()
#5 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/framework/web/CWebApplication.php(282): CController->run('novoUsuario')
#9 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/novoUsuari...')
#10 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#11 /var/www/feitobrick/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=site/novoUsuario
HTTP_REFERER=http://localhost:100/index.php?r=site/novoUsuario
---
2013/02/09 17:04:44 [error] [exception.CException] exception 'CException' with message 'Property "novoUsuarioForm.bairro" is not defined.' in /var/www/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/framework/validators/CStringValidator.php(79): CComponent->__get('bairro')
#1 /var/www/framework/validators/CValidator.php(213): CStringValidator->validateAttribute(Object(novoUsuarioForm), 'bairro')
#2 /var/www/framework/base/CModel.php(159): CValidator->validate(Object(novoUsuarioForm), NULL)
#3 /var/www/framework/web/widgets/CActiveForm.php(835): CModel->validate(NULL)
#4 /var/www/feitobrick/protected/components/GxController.php(168): CActiveForm::validate(Object(novoUsuarioForm))
#5 /var/www/feitobrick/protected/controllers/SiteController.php(25): GxController->performAjaxValidation(Object(novoUsuarioForm), 'usuario-form')
#6 /var/www/framework/web/actions/CInlineAction.php(49): SiteController->actionNovoUsuario()
#7 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/framework/web/CWebApplication.php(282): CController->run('novoUsuario')
#11 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/novoUsuari...')
#12 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#13 /var/www/feitobrick/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/index.php?r=site/novoUsuario
HTTP_REFERER=http://localhost:100/index.php?r=site/novoUsuario
---
2013/02/09 17:05:08 [error] [exception.CException] exception 'CException' with message 'Property "Usuario.ativo" is not defined.' in /var/www/framework/base/CComponent.php:173
Stack trace:
#0 /var/www/framework/db/ar/CActiveRecord.php(159): CComponent->__set('ativo', 't')
#1 /var/www/feitobrick/protected/controllers/SiteController.php(35): CActiveRecord->__set('ativo', 't')
#2 /var/www/framework/web/actions/CInlineAction.php(49): SiteController->actionNovoUsuario()
#3 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#5 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /var/www/framework/web/CWebApplication.php(282): CController->run('novoUsuario')
#7 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/novoUsuari...')
#8 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#9 /var/www/feitobrick/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=site/novoUsuario
HTTP_REFERER=http://localhost:100/index.php?r=site/novoUsuario
---
2013/02/09 17:05:50 [error] [exception.CException] exception 'CException' with message 'Property "Usuario.numrua" is not defined.' in /var/www/framework/base/CComponent.php:173
Stack trace:
#0 /var/www/framework/db/ar/CActiveRecord.php(159): CComponent->__set('numrua', '87')
#1 /var/www/feitobrick/protected/controllers/SiteController.php(43): CActiveRecord->__set('numrua', '87')
#2 /var/www/framework/web/actions/CInlineAction.php(49): SiteController->actionNovoUsuario()
#3 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#5 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /var/www/framework/web/CWebApplication.php(282): CController->run('novoUsuario')
#7 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/novoUsuari...')
#8 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#9 /var/www/feitobrick/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=site/novoUsuario
HTTP_REFERER=http://localhost:100/index.php?r=site/novoUsuario
---
2013/02/09 17:06:36 [warning] [application] Failed to set unsafe attribute "num" of "novoUsuarioForm".
in /var/www/feitobrick/protected/controllers/SiteController.php (29)
in /var/www/feitobrick/index.php (13)
2013/02/09 17:06:36 [error] [exception.CException] exception 'CException' with message 'Property "novoUsuarioForm.num" is not defined.' in /var/www/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/framework/web/helpers/CHtml.php(2255): CComponent->__get('num')
#1 /var/www/framework/web/helpers/CHtml.php(1980): CHtml::resolveValue(Object(novoUsuarioForm), 'num')
#2 /var/www/framework/web/helpers/CHtml.php(1257): CHtml::activeInputField('text', Object(novoUsuarioForm), 'num', Array)
#3 /var/www/framework/web/widgets/CActiveForm.php(657): CHtml::activeTextField(Object(novoUsuarioForm), 'num', Array)
#4 /var/www/feitobrick/protected/views/site/novoUsuario.php(154): CActiveForm->textField(Object(novoUsuarioForm), 'num', Array)
#5 /var/www/framework/web/CBaseController.php(126): require('/var/www/feitob...')
#6 /var/www/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/feitob...', Array, true)
#7 /var/www/framework/web/CController.php(869): CBaseController->renderFile('/var/www/feitob...', Array, true)
#8 /var/www/framework/web/CController.php(782): CController->renderPartial('novoUsuario', Array, true)
#9 /var/www/feitobrick/protected/controllers/SiteController.php(55): CController->render('novoUsuario', Array)
#10 /var/www/framework/web/actions/CInlineAction.php(49): SiteController->actionNovoUsuario()
#11 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#12 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#13 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 /var/www/framework/web/CWebApplication.php(282): CController->run('novoUsuario')
#15 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/novoUsuari...')
#16 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#17 /var/www/feitobrick/index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/index.php?r=site/novoUsuario
HTTP_REFERER=http://localhost:100/index.php?r=site/novoUsuario
---
2013/02/09 17:06:55 [warning] [application] Failed to set unsafe attribute "num" of "novoUsuarioForm".
in /var/www/feitobrick/protected/controllers/SiteController.php (29)
in /var/www/feitobrick/index.php (13)
2013/02/09 17:07:11 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23502]: Not null violation: 7 ERRO:  valor nulo na coluna "idusuario" viola a restrição não-nula
DETAIL:  Failing row contains (null, iury, 898.989.898-98, 9898989898, , 1298-12-12, iurycl@gmail.com, dd6db1e56bc6befc2160940048ca6bf2, iuiu, iuiu, AC).. The SQL statement executed was: INSERT INTO "usuario" ("nome", "datanasc", "rg", "cpf", "email", "senha", "estado", "cidade", "rua", "telefone") VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9).
in /var/www/feitobrick/protected/controllers/SiteController.php (45)
in /var/www/feitobrick/index.php (13)
2013/02/09 17:07:11 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23502]: Not null violation: 7 ERRO:  valor nulo na coluna "idusuario" viola a restrição não-nula
DETAIL:  Failing row contains (null, iury, 898.989.898-98, 9898989898, , 1298-12-12, iurycl@gmail.com, dd6db1e56bc6befc2160940048ca6bf2, iuiu, iuiu, AC).. The SQL statement executed was: INSERT INTO "usuario" ("nome", "datanasc", "rg", "cpf", "email", "senha", "estado", "cidade", "rua", "telefone") VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9)' in /var/www/framework/db/CDbCommand.php:357
Stack trace:
#0 /var/www/framework/db/ar/CActiveRecord.php(1023): CDbCommand->execute()
#1 /var/www/framework/db/ar/CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 /var/www/feitobrick/protected/controllers/SiteController.php(45): CActiveRecord->save(false)
#3 /var/www/framework/web/actions/CInlineAction.php(49): SiteController->actionNovoUsuario()
#4 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#6 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /var/www/framework/web/CWebApplication.php(282): CController->run('novoUsuario')
#8 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/novoUsuari...')
#9 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#10 /var/www/feitobrick/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=site/novoUsuario
HTTP_REFERER=http://localhost:100/index.php?r=site/novoUsuario
---
2013/02/09 17:08:19 [error] [php] Undefined variable: duration (/var/www/feitobrick/protected/models/LoginForm.php:63)
Stack trace:
#0 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams()
#1 /var/www/framework/web/CController.php(286): SiteController->runAction()
#2 /var/www/framework/web/CController.php(265): SiteController->runActionWithFilters()
#3 /var/www/framework/web/CWebApplication.php(282): SiteController->run()
#4 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController()
#5 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#6 /var/www/feitobrick/index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=site/novoUsuario
in /var/www/feitobrick/protected/models/LoginForm.php (63)
in /var/www/feitobrick/protected/controllers/SiteController.php (50)
in /var/www/feitobrick/index.php (13)
2013/02/09 17:10:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "cadastro".' in /var/www/framework/web/CController.php:483
Stack trace:
#0 /var/www/framework/web/CController.php(270): CController->missingAction('cadastro')
#1 /var/www/framework/web/CWebApplication.php(282): CController->run('cadastro')
#2 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/cadastro')
#3 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/feitobrick/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/cadastro
HTTP_REFERER=http://localhost:100/index.php
---
2013/02/09 17:14:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "meusProdutos".' in /var/www/framework/web/CController.php:483
Stack trace:
#0 /var/www/framework/web/CController.php(270): CController->missingAction('meusProdutos')
#1 /var/www/framework/web/CWebApplication.php(282): CController->run('meusProdutos')
#2 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/meusProdut...')
#3 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/feitobrick/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/meusProdutos
HTTP_REFERER=http://localhost:100/index.php?r=site/contato
---
2013/02/09 17:15:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "meusProdutos".' in /var/www/framework/web/CController.php:483
Stack trace:
#0 /var/www/framework/web/CController.php(270): CController->missingAction('meusProdutos')
#1 /var/www/framework/web/CWebApplication.php(282): CController->run('meusProdutos')
#2 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/meusProdut...')
#3 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/feitobrick/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/meusProdutos
HTTP_REFERER=http://localhost:100/index.php?r=site/contato
---
2013/02/11 16:20:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "meusProdutos".' in /var/www/framework/web/CController.php:483
Stack trace:
#0 /var/www/framework/web/CController.php(270): CController->missingAction('meusProdutos')
#1 /var/www/framework/web/CWebApplication.php(282): CController->run('meusProdutos')
#2 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/meusProdut...')
#3 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/feitobrick/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/meusProdutos
HTTP_REFERER=http://localhost:100/index.php
---
2013/02/13 20:37:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "meusProdutos".' in /var/www/framework/web/CController.php:483
Stack trace:
#0 /var/www/framework/web/CController.php(270): CController->missingAction('meusProdutos')
#1 /var/www/framework/web/CWebApplication.php(282): CController->run('meusProdutos')
#2 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/meusProdut...')
#3 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/feitobrick/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/meusProdutos
HTTP_REFERER=http://localhost:100/index.php
---
2013/02/14 12:07:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "meusProdutos".' in /var/www/framework/web/CController.php:483
Stack trace:
#0 /var/www/framework/web/CController.php(270): CController->missingAction('meusProdutos')
#1 /var/www/framework/web/CWebApplication.php(282): CController->run('meusProdutos')
#2 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/meusProdut...')
#3 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/feitobrick/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/meusProdutos
HTTP_REFERER=http://localhost:100/index.php
---
2013/02/14 12:17:14 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "produto1".' in /var/www/framework/web/CController.php:878
Stack trace:
#0 /var/www/framework/web/CController.php(782): CController->renderPartial('produto1', NULL, true)
#1 /var/www/feitobrick/protected/controllers/SiteController.php(26): CController->render('produto1')
#2 /var/www/framework/web/actions/CInlineAction.php(49): SiteController->actionProduto1()
#3 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#5 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /var/www/framework/web/CWebApplication.php(282): CController->run('produto1')
#7 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/produto1')
#8 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#9 /var/www/feitobrick/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=site/produto1
HTTP_REFERER=http://localhost:100/index.php?r=site/produtos
---
2013/02/14 12:18:14 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "produto2".' in /var/www/framework/web/CController.php:878
Stack trace:
#0 /var/www/framework/web/CController.php(782): CController->renderPartial('produto2', NULL, true)
#1 /var/www/feitobrick/protected/controllers/SiteController.php(32): CController->render('produto2')
#2 /var/www/framework/web/actions/CInlineAction.php(49): SiteController->actionProduto2()
#3 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#5 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /var/www/framework/web/CWebApplication.php(282): CController->run('produto2')
#7 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/produto2')
#8 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#9 /var/www/feitobrick/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=site/produto2
HTTP_REFERER=http://localhost:100/index.php?r=site/produtos
---
2013/02/14 12:24:09 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "produto3".' in /var/www/framework/web/CController.php:878
Stack trace:
#0 /var/www/framework/web/CController.php(782): CController->renderPartial('produto3', NULL, true)
#1 /var/www/feitobrick/protected/controllers/SiteController.php(38): CController->render('produto3')
#2 /var/www/framework/web/actions/CInlineAction.php(49): SiteController->actionProduto3()
#3 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#5 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /var/www/framework/web/CWebApplication.php(282): CController->run('produto3')
#7 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/produto3')
#8 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#9 /var/www/feitobrick/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=site/produto3
HTTP_REFERER=http://localhost:100/index.php?r=site/produtos
---
2013/02/14 12:32:06 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "produto3".' in /var/www/framework/web/CController.php:878
Stack trace:
#0 /var/www/framework/web/CController.php(782): CController->renderPartial('produto3', NULL, true)
#1 /var/www/feitobrick/protected/controllers/SiteController.php(38): CController->render('produto3')
#2 /var/www/framework/web/actions/CInlineAction.php(49): SiteController->actionProduto3()
#3 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#5 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /var/www/framework/web/CWebApplication.php(282): CController->run('produto3')
#7 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/produto3')
#8 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#9 /var/www/feitobrick/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=site/produto3
HTTP_REFERER=http://localhost:100/index.php?r=site/produtos
---
2013/02/14 12:49:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "meusProdutos".' in /var/www/framework/web/CController.php:483
Stack trace:
#0 /var/www/framework/web/CController.php(270): CController->missingAction('meusProdutos')
#1 /var/www/framework/web/CWebApplication.php(282): CController->run('meusProdutos')
#2 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/meusProdut...')
#3 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/feitobrick/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/meusProdutos
HTTP_REFERER=http://localhost:100/index.php
---
2013/02/14 13:02:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "meusProdutos".' in /var/www/framework/web/CController.php:483
Stack trace:
#0 /var/www/framework/web/CController.php(270): CController->missingAction('meusProdutos')
#1 /var/www/framework/web/CWebApplication.php(282): CController->run('meusProdutos')
#2 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/meusProdut...')
#3 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/feitobrick/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/meusProdutos
HTTP_REFERER=http://localhost:100/index.php
---
2013/02/15 19:38:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "meusProdutos".' in /var/www/framework/web/CController.php:483
Stack trace:
#0 /var/www/framework/web/CController.php(270): CController->missingAction('meusProdutos')
#1 /var/www/framework/web/CWebApplication.php(282): CController->run('meusProdutos')
#2 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/meusProdut...')
#3 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/feitobrick/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/meusProdutos
HTTP_REFERER=http://localhost:100/index.php
---
2013/02/15 19:38:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "meusProdutos".' in /var/www/framework/web/CController.php:483
Stack trace:
#0 /var/www/framework/web/CController.php(270): CController->missingAction('meusProdutos')
#1 /var/www/framework/web/CWebApplication.php(282): CController->run('meusProdutos')
#2 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/meusProdut...')
#3 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/feitobrick/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/meusProdutos
HTTP_REFERER=http://localhost:100/index.php?r=site/meusProdutos
---
2013/02/15 20:29:06 [error] [php] include(novoProdutoForm.php): failed to open stream: No such file or directory (/var/www/framework/YiiBase.php:421)
Stack trace:
#0 /var/www/feitobrick/protected/controllers/SiteController.php(93): spl_autoload_call()
#1 /var/www/framework/web/actions/CInlineAction.php(49): SiteController->actionNovoProduto()
#2 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams()
#3 /var/www/framework/web/CController.php(286): SiteController->runAction()
#4 /var/www/framework/web/CController.php(265): SiteController->runActionWithFilters()
#5 /var/www/framework/web/CWebApplication.php(282): SiteController->run()
#6 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController()
#7 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#8 /var/www/feitobrick/index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=site/novoProduto
in /var/www/feitobrick/protected/controllers/SiteController.php (93)
in /var/www/feitobrick/index.php (13)
